pidTuner.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <vector>
+
+class pidTuner {
+public:
+
+    pidTuner(float ip, float ii, float id, float Sp=.5, float Si=.05, float Sd=.01);
+
+    void start_cycle();
+    void run(float err);
+    bool end_cycle();
```

Comment:
I would add some doxygen comments to these to describe their usage. Something like the order to call them and how many times.